cmd/build.sh

```diff
@@ -125,24 +125,35 @@ do
 
     echo -en "\t - $goos/$goarch..."
 
-    zipfile="$filename-$build_version-$pos-$parch"
-    zipdir="${buildPath}/$filename"
-    mkdir -p $zipdir
+    mi_dir_name="wso2$filename-cli-$build_version"
+    mi_archive_name="$mi_dir_name-$pos-$parch"
+    mi_archive_dir="${buildPath}/${mi_dir_name}"
+    mkdir -p $mi_archive_dir
 
-    cp -r "${baseDir}/server_config.yaml" $zipdir > /dev/null 2>&1
-    cp -r "${baseDir}/LICENSE" $zipdir > /dev/null 2>&1
+    cp -r "${baseDir}/docs/README.md" $mi_archive_dir > /dev/null 2>&1
```

Comment:
Shall we add the README.md to root directory instead of in a docs folder similar to License file?

Reply:
In the distribution, it's added to the root directory
.
├── bin
│   └── mi
└── LICENSE
└── README.md
└── mi_bash_completion.sh